Makefile

```
@@ -1,6 +1,8 @@
 # Prepare variables
 TMP = $(CURDIR)/tmp
 VERSION = $(shell grep ^Version tmt.spec | sed 's/.* //')
+COMMIT = $(shell git rev-parse --short HEAD)
+REPLACE_VERSION = "s/running from the source/$(VERSION) ($(COMMIT))/"
```

```suggestion
REPLACE_VERSION = "/^VERSION =/ s/running from the source/$(VERSION) ($(COMMIT))/"
```

Optional enhancement of the `sed`.

---

Good idea, @lukaszachy. Moved the version to `tmt.__version__`.